src/clj/game/cards/programs.clj

```diff
@@ -598,20 +598,20 @@
     :abilities [{:cost [:credit 1] :msg (msg "break 1 " (:subtype-target card) " subroutine")}]}
 
    "Chisel"
-   {:effect (effect (system-say "Corp, please press No Action to trigger Chisel"))
+   {:implementation "Encounter effect is manual."
     :hosting {:req (every-pred ice? can-host?)}
+    :abilities [{:req (req (= (:cid current-ice) (:cid (:host card))))
+                 :msg (msg (if (zero? (get-strength current-ice))
```

Comment:
What is the strength of an un-rezzed ice? Will this cause ice to be trashed if used erroneously on un-rezzed ice?

Reply:
It calls `(or (:current-strength card) (:strength card))`, so it'll always populate with the "correct" number.

Reply:
Hmm, maybe the best way to handle unrezzed ice is to add `(rezzed? current-ice)` to the req (if that isn't already handled somehow with how current-ice is bound)

Reply:
Oh I see what you mean. Good call, I'll add a check for the ice being rezzed.